sdk/executor/src/main/java/com/mesosphere/sdk/executor/CustomExecutor.java

```diff
@@ -44,15 +43,13 @@ public void registered(
             Protos.ExecutorInfo executorInfo,
             Protos.FrameworkInfo frameworkInfo,
             Protos.SlaveInfo slaveInfo) {
-        LOGGER.info("Registered executor: {}", executorInfo.getExecutorId());
+        LOGGER.info("Registered executor: {}", TextFormat.shortDebugString(executorInfo));
         this.slaveInfo = slaveInfo;
-        this.executorInfo = executorInfo;
-        this.frameworkInfo = frameworkInfo;
     }
 
     @Override
     public void reregistered(ExecutorDriver driver, Protos.SlaveInfo slaveInfo) {
-        LOGGER.info("Re-registered on slave: {}", slaveInfo.getId());
+        LOGGER.info("Re-registered on slave: {}", TextFormat.shortDebugString(slaveInfo));
```

Comment:
This might be a case where I'd prefer just the ID, since the rest of the agent info isn't really relevant, can be cross-referenced anyway, and there are probably other logs that contain the meat of the issue that I wouldn't want obscured by this full proto debug string. What do you think?